frontend/src/css/FeedItemWithPreview.styl

```diff
@@ -29,5 +30,5 @@ smallScreenFeedWidth = 400px
 
 
 @media (max-width: max-sm)
-  .feedItem
-    width smallScreenFeedWidth
+  .feedItem__timeline__texteditor__advancedtext, .publications__publishArea__buttons__advancedEdition
```

Comment:
I would prefer a component to not interact with the style of another component
couldn't you put the `.publications__publishArea__buttons__advancedEdition` in its own css file?

Reply:
oh, yes, indeed

Reply:
fixed